kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/index.vue

```diff
@@ -351,7 +341,7 @@
           });
       },
       detailLink(learnerId) {
-        return this.classRoute(PageNames.QUIZ_LEARNER_PAGE_ROOT, {
+        return this.classRoute(this.PageNames.QUIZ_LEARNER_PAGE_ROOT, {
```

Comment:
This was weird to find but it's because we don't import PageNames here, but rather, we get it from the `commonCoach` mixin